fpga/src/main/resources/vcu118/sdboot/sd.c

```diff
@@ -168,9 +170,11 @@ static int copy(void)
 	int rc = 0;
 
 	dputs("CMD18");
+
+	kprintf("LOADING 0x%xB PAYLOAD\r\n", PAYLOAD_SIZE_B);
 	kprintf("LOADING  ");
```

Comment:
Why do we need both of these messages? The second loading seems redundant.

Reply:
There is a nice spinner that comes up next to the loading so I just left this. Technically I can put the spinner next to the `kprintf` that I added, but I figured this extra print wasn't a bottleneck so just leave it.